.github/workflows/build.yaml

```diff
@@ -0,0 +1,22 @@
+name: Trigger Helm workflow in automation repository
+
+on:
+  push:
+    branches: [main]
+  
+env:
+  NAMESPACE: galasa-dev
+
+jobs:
+    trigger-workflow:
+        name: trigger helm workflow
+        runs-on: ubuntu-latest
+        permissions: write-all
+
+        steps:
+        - name: Triggering workflow using Github API call
+          run: |
+           gh workflow run build-helm.yaml --repo https://github.com/${{env.NAMESPACE}}/automation
+          env:
+            GH_TOKEN: ${{ github.token }}

```

Comment:
Please change this to `${{ secrets.GALASA_TEAM_GITHUB_TOKEN }}` I believe this is needed for cross repo triggering, where `github.token` doesn't have the necessary permissions.

Reply:
done

Reply:
probably the reason this was failing.